### ISSUE TITLE
Error "startswith first arg must be str or a tuple of str, not list"

### ISSUE THREAD

**Issue by Ilocans** [maykar/plex_assistant#28]
Hi,

I tested plex_assistant.command service found in Developer Tools > Services with the example and I have the following error: "Failed to call service plex_assistant/command. startswith first arg must be str or a tuple of str, not list"

I installed plex_assistant using hacs and I am on freenas.

Thanks

**Comment by maykar** [comment]
Do you mean you used the default example by hitting "Fill Example Data"? So, like this:
```
command: Play Evil Dead on the Downstairs TV
cast_delay: 6
```

Or did you send a different command and if so, what?

**Comment by maykar** [comment]
Also could you paste the whole error (click on it and copy paste the result)?

**Comment by rippergun** [comment]
Hi, same problem here with the log
```Log Details (ERROR)
Logger: homeassistant.components.websocket_api.http.connection.140536386052304
Source: custom_components/plex_assistant/process_speech.py:20
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 8:54:09 AM (2 occurrences)
Last logged: 8:54:16 AM

startswith first arg must be str or a tuple of str, not list
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 130, in handle_call_service
    connection.context(msg),
  File "/usr/src/homeassistant/homeassistant/core.py", line 1260, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1299, in _execute_service
    await self._hass.async_add_executor_job(handler.func, service_call)
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/plex_assistant/__init__.py", line 109, in handle_input
    PA
  File "/config/custom_components/plex_assistant/process_speech.py", line 20, in process_speech
    if command.startswith(controls[control]):
TypeError: startswith first arg must be str or a tuple of str, not list```

**Comment by maykar** [comment]
Sorry for the delay, this was an error in the French translation. Should be fixed in latest version ( 0.1.8 )
